toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -50,12 +51,15 @@ type PackageList struct {
 // GetRequiredPackagesForInstall returns the list of packages required for
 // the tooling to install an image
 func GetRequiredPackagesForInstall() []*pkgjson.PackageVer {
+	requiredPackageNames := []string{}
+
 	// grub2-pc package is needed for the install tools to build/install the legacy grub bootloader
-	requiredPackageNames := []string{
-		"grub2-pc",
+	// Note: only required on x86_64 installs
+	if runtime.GOARCH == "amd64" {
+		requiredPackageNames = append(requiredPackageNames, "grub2-pc")
 	}
 
-	var packageList []*pkgjson.PackageVer
+	packageList := []*pkgjson.PackageVer{}
```

Comment:
When I think about it, I'm curious if we could start with an empty array of `pkgjson.PackageVer` immediately and simply drop `requiredPackageNames`. After all in both cases we're just appending to a list.

Reply:
Agree with this simplification.